custom-words.txt

```diff
@@ -2484,6 +2484,22 @@ loadtest
 xvalue
 yvalue
 servermetrics
+napt
+Napt
+NAPT
+Plmn
+IMSI
+UE's
+Preemptable
+Ambr
+AMBR
+rfsp
+Rfsp
+snssai
+Snssai
+NSSAI
+Tbps
+Vuln
```

Comment:
I think these are already in this list a few lines below so you can remove them.

Reply:
Addressed.